src/EprPrnIntegration.Common/Models/Npwd/Producer.cs

```diff
@@ -5,49 +5,49 @@
 public class Producer
 {
     [JsonPropertyName("AddressLine1")]
-    public string AddressLine1 { get; set; }
+    public string AddressLine1 { get; set; } = default!;
 
     [JsonPropertyName("AddressLine2")]
-    public string AddressLine2 { get; set; }
+    public string AddressLine2 { get; set; } = default!;
 
     [JsonPropertyName("AddressLine3")]
-    public string AddressLine3 { get; set; }
+    public string AddressLine3 { get; set; } = default!;
```

Comment:
Will this always present can we confirm it won't be null. I believe default! will fail if it don't get any value when object creation. Same with Address line 4 as well